### PR TITLE
fix: container input color

### PR DESCRIPTION
**Pull request by kukac7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [3.0.9](https://github.com/ng-select/ng-select/compare/v3.0.8...v3.0.9) (2019-09-02)
```
Comment (anjmao) [inline CHANGELOG.md]: This change is not needed. Changelog is updated when doing a new release.

File: src/ng-select/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ng-select/ng-select",
-  "version": "3.0.7",
+  "version": "3.0.9",
```
Comment (anjmao) [inline src/ng-select/package.json]: This change is not needed.

Comment (kukac7) [reply src/ng-select/package.json]: @anjmao my fault. sorry

File: CHANGELOG.md (hunk 1 above)
Comment (kukac7) [reply CHANGELOG.md]: @anjmao my fault. sorry

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
```
Comment (varnastadeus) [inline CHANGELOG.md]: Please remove any changes from changelog

Comment (kukac7) [reply CHANGELOG.md]: @varnastadeus will be good?

**Pull request closed without merging**